packages/account/src/Styles/account.scss

```diff
@@ -1182,3 +1182,201 @@ $MIN_HEIGHT_FLOATING: calc(
+                font-weight: normal;
+            }
+        }
+        &__input {
+            margin: 16px 0 0;
+            max-width: 400px;
+            text-overflow: ellipsis;
+
+            > input {
+                min-height: 96px;
+            }
+        }
+        &__error {
+            color: var(--text-loss-danger);
+        }
+    }
+    @include mobile {
+        width: 92%;
+        margin: 0 auto;
+    }
+}
+
+/* stylelint-disable */
+.dc-modal__container_deactivate-account-reasons {
+    .dc-modal-header {
+        border: none;
+        min-width: 430px;
+    }
+    .account-closure-warning-modal {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        padding: 3.2rem;
+
+        .dc-form-submit-button {
+            justify-content: center;
+            padding: 2rem 0 0;
+
+            @include mobile {
+                background: unset;
+            }
+        }
+        &__warning-message {
+            color: var(--text-loss-danger);
+            padding: 16px 0;
+            font-size: var(--text-size-s);
+            font-weight: bold;
+            line-height: 1.43;
+        }
+        &__content {
+            @include typeface(--paragraph-left-normal-prominent, none);
+            line-height: 1.43;
+        }
+        &__content-wrapper {
+            display: flex;
+            align-items: center;
+            margin-top: 4px;
+        }
+    }
+    .have-open-positions {
+        padding: 24px;
+
+        &__title {
+            color: var(--text-prominent);
+            font-size: var(--text-size-xs);
+            font-weight: bold;
+        }
+        &__accounts-wrapper {
+            display: flex;
+            padding: 8px 8px 8px 16px;
+            align-items: center;
+        }
+        &__accounts-data {
+            display: flex;
+            flex-direction: column;
+            margin-left: 8px;
+        }
+        &__accounts-loginid {
+            color: var(--text-prominent);
+            font-size: var(--text-size-xxxs);
+            line-height: 1.4;
+        }
+        &__accounts-currency {
+            color: var(--text-prominent);
+            font-size: var(--text-size-xs);
+            line-height: 1.43;
+        }
+        &__accounts-button {
+            margin-top: 16px;
+            float: right;
+        }
+    }
+    .existing-account-has-balance {
+        padding: 0 24px 24px 24px;
+
+        &__action {
+            padding: 8px 0;
+            color: var(--text-prominent);
+            font-size: var(--text-size-xs);
+            line-height: 1.43;
+            font-weight: bold;
+        }
+        &__container {
+            display: flex;
+            padding: 3px 16px;
+            justify-content: space-between;
+            align-items: center;
+
+            &__account-details {
+                display: flex;
+                align-items: center;
+
+                &__balance {
```

Comment:
(throughout)
Nested class reference is against BEM. You can use `.existing-account-has-balance__balance` and move this up.

Reply:
done